pkg/client/node.go

```diff
@@ -329,6 +333,34 @@ func NodeCreate(ctx context.Context, runtime runtimes.Runtime, node *k3d.Node, c
 		}
 	}
 
+	// memory limits
+	if node.Memory != "" {
+		memory, err := dockerunits.RAMInBytes(node.Memory)
+		if err != nil {
+			return fmt.Errorf("Invalid memory limit format: %+v", err)
+		}
+		// mount fake meminfo as readonly
+		fakemempath, err := util.MakeFakeMeminfo(memory, node.Name)
+		if err != nil {
+			return fmt.Errorf("Failed to create fake meminfo: %+v", err)
+		}
+		node.Volumes = append(node.Volumes, fmt.Sprintf("%s:/proc/meminfo:ro", fakemempath))
+		// mount empty edac folder, but only if it exists
+		exists, err := docker.CheckIfDirectoryExists(ctx, node.Image, "/sys/devices/system/edac")
```

Comment:
Only, if `runtime == runtimes.Docker`, right?

Reply:
Also, you may want to make that path a constant value somewhere

Reply:
Yup, this whole implementation is valid for docker only.